components/Common/ProgressionSidebar/ProgressionSidebarGroup/index.module.css

```diff
@@ -0,0 +1,61 @@
+.group,
```

Comment:
Is there any reason why this styles both .group and the root ul? Afaik it has no effect a root `ul` style here as this is all sandboxes and only classnames actually get exported from CSS modules